src/adapters/sequelize.js

```diff
@@ -82,8 +83,16 @@ module.exports = (model, opts) => {
         idField = 'forestCompositePrimary';
       }
 
+      Object.entries(model.associations).forEach(([, association]) => {
+        const pkIsFk = primaryKeyIsForeignKey(association);
+        if (pkIsFk) {
+          const fk = fields.find((field) => field.reference === `${association.associationAccessor}.${association.foreignKey}`);
+          fk.fkIsPk = true;
+        }
+      });
+
       _.remove(fields, (field) =>
-        _.includes(fieldNamesToExclude, field.columnName) && !field.primaryKey);
```

Comment:
`primaryKey` does not exist, but `isPrimaryKey` does.
Neithertheless, we still need to remove it for a better user experience.